docs/src/tutorials/applications/web_app.jl

```diff
@@ -0,0 +1,141 @@
+# Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors    #src
+# This Source Code Form is subject to the terms of the Mozilla Public License   #src
+# v.2.0. If a copy of the MPL was not distributed with this file, You can       #src
+# obtain one at https://mozilla.org/MPL/2.0/.                                   #src
+
+# # Serving web apps
+
+# This tutorial demonstrates how to setup and serve JuMP models via a REST API.
```

Comment:
Maybe this could be made slightly clearer for beginners by saying how the parameters and solutions are transferred. I would add the following: 

> In the example app we are building, we are interested in solving a trivial mixed-integer program with one parameter, a lower bound. To call the service, the users are expected to send an HTTP POST request with JSON contents indicating the lower bound. The returned value is the solution of the MIP as JSON.